coalib/testing/LocalBearTestHelper.py

```diff
@@ -25,6 +25,22 @@ def execute_bear(bear, *args, **kwargs):
                 console_output.append('The program yielded '
                                       'the following output:\n')
                 old_process_output = bear.process_output
+                old_create_arguments = bear.create_arguments
+
+                def new_create_arguments(filename,
+                                         file,
+                                         config_file,
+                                         *args,
+                                         **kwargs):
+                    arguments = old_create_arguments(filename,
+                                                     file,
+                                                     config_file,
+                                                     *args,
+                                                     **kwargs)
+                    console_output.append(
+                        'Arguments:\n' + (' ').join(arguments))
```

Comment:
Ah maybe instead of joining, just append `repr(arguments)`, then it appears like python code, which probably eases debugging in many cases. Especially `\n` would be escaped then.